autopush/jwt.py

```diff
@@ -0,0 +1,84 @@
+"""Why hand roll?
```

Comment:
This will show up as the module's title in the API docs.... oh, maybe we should add a .rst so that there'll be docs generated for this module?